lib/ziti-tunnel/lwip/netif_shim.c

```diff
@@ -70,17 +70,25 @@ void netif_shim_input(struct netif *netif) {
 }
 
 void on_packet(const char *buf, ssize_t nr, void *ctx) {
+    static bool log_pbuf_errors = true;
     struct netif *netif = ctx;
     struct pbuf *p;
     /* We allocate a pbuf chain of pbufs from the pool. */
     p = pbuf_alloc(PBUF_LINK, (u16_t) nr, PBUF_POOL);
 
     if (p != NULL) {
         pbuf_take(p, buf, (u16_t) nr);
+        if (!log_pbuf_errors) {
```

Comment:
ancillary question to the PR: shouldn't the `pbuf_take` above be checking for errors too?